README.md

```diff
@@ -1,92 +1,38 @@
+<img src="https://img.shields.io/badge/-CSESoc-blue" />
+
 # Circles
+> see the [confluence workspace and links](https://compclub.atlassian.net/wiki/spaces/C/pages/1664385928/About+Circles) for further development documentation and a linktree to our jira boards, and more!
+## about Circles
+Tired of using a poorly laid out spreadsheet to cobble together a plan to follow for the next 3-8 years of your life?
+have no fear! Circles is here!
+
+Circles is a UNSW degree planner, which you can use to explore, check, and play around with your course planning.
```

Comment:
Not sure how to reword this, maybe something like 'Circles is a UNSW degree planner where you can explore, validate and tinker with your course structure.', not sure about the with your course structure bit...